CWE/events/italy.txt

```diff
@@ -54,460 +126,341 @@ country_event = {
+	desc = EVTDESC8007772
+	picture = "italian_parliament"
   
 	fire_only_once = yes
-  
+
 	trigger = {
 		tag = ITA
-		has_global_flag = blocsenabled
-		ruling_party_ideology = communist
-		government = proletarian_dictatorship
-		election = no
-	}
-
-	option = {
-		name = EVT_8007743_A
-		
-		#Break alliances with non communists
-		any_country = {
-			limit = { NOT = { ruling_party_ideology = communist } }
-			leave_alliance = THIS
-			relation = { who = THIS value = -200 }
-		}
-		
-		#Be friends with fellow communists
-		any_country = {
-			limit = { ruling_party_ideology = communist }
-			relation = { who = THIS value = 200 }
-		}
+		year = 1948
+		NOT = { year = 1990 }
 		
-		#Citizens become more communist
-		any_pop = {
-			limit = { is_primary_culture = yes }
-			ideology = { factor = 0.05 value = communist }
-		}
+		government = democracy
+		is_vassal = no
+		war = no
 		
-		#USA choice to declare war
-		USA = { country_event = 8007744 }
+		AND = { has_country_flag = 1948_ITA_FDP has_country_flag = ITA_Togliatti_shooting }
+		NOT = { has_country_flag = 1948_ITA_Centrism }
 	}
 
-}
-
-#Hardline communists gain power in Italy
-country_event = {
-  id = 8007744
-  title = EVT_8007744_NAME
-  desc = EVT_8007744_DESC
-  picture = "nwo2_italian_communist_party"
-  is_triggered_only = yes # USA
-
-	option = {
-		name = "It was the choice of the Italian people"
-		RUS = { country_event = 8007745 } #Victory for the Russians
-		ai_chance = { factor = 0.6 }
-	}
+	mean_time_to_happen = { months = 4 }
 
 	option = {
-		name = "We have to intervene!"
-		
-		war = { 
-			target = ITA 
-			attacker_goal = { casus_belli = install_democracy } 
-			defender_goal = { casus_belli = humiliate } 
-		}
-		
-		#USSR choice to defend
+		name = "Enact the new laws!"
 		random_country = {
-			limit = { tag = RUS government = proletarian_dictatorship }
-			country_event = 8007745 #RUS choice to ally with ITA
-		}
+                       limit = { tag = USA government = democracy is_vassal = no }
+                       relation = { who = THIS value = -100 }
+        }
+		random_country = {
+                       limit = { tag = ENG government = democracy is_vassal = no }
+                       relation = { who = THIS value = -100 }
+        }
+		random_country = {
+                       limit = { tag = FRA government = democracy is_vassal = no }
+                       relation = { who = THIS value = -100 }
+        }
+		random_country = {
+                       limit = { tag = RUS government = proletarian_dictatorship is_vassal = no }
+                       relation = { who = THIS value = 50 }
+        }
 		
-		ai_chance = { factor = 0.4 }
-	}
-
-}
-
-#RUS choice to defend Communist Italy
-country_event = {
-  id = 8007745
-  title = EVT_8007745_NAME
-  desc = EVT_8007745_DESC
-  picture = "nwo2_italian_communist_party"
-  is_triggered_only = yes # RUS
+		government = proletarian_dictatorship
+      
+		political_reform = regulated_opinion
```

Comment:
It might be better to use country_event = 800052 to set the reforms when setting up a communist state.